arbitrum-docs/how-arbitrum-works/10-l1-to-l2-messaging.mdx

```diff
@@ -1,147 +1,437 @@
 ---
-title: L1 to L2 messaging
-description: 'Learn the fundamentals of L1 to L2 messaging on Arbitrum.'
+title: Bridging from a parent to a child chain
+description: 'Learn the fundamentals of parent to child chain messaging on Arbitrum.'
 author: pete-vielhaber
-sme: TucksonDev
-user_story: As a current or prospective Arbitrum user, I need to learn more about messaging between L1 and L2 within Arbitrum.
+sme: Mehdi
+user_story: As a current or prospective Arbitrum user, I need to learn more about messaging between a parent and child chain within Arbitrum.
 content_type: get-started
 ---
 
-import { AddressAliasHelper } from '@site/src/components/AddressAliasHelper';
+In the [Bypassing the Sequencer] section, we introduced an alternative way for users to submit transactions to a child chain by going through the parent chain's delayed inbox contract instead of sending them directly to the Sequencer. This approach is one example of a parent to child messaging path. More broadly, parent to child chain messaging covers all ways to:
```

Comment:
```suggestion
In the [Bypassing the Sequencer] section, we introduced an alternative way for users to submit transactions to a child chain by going through the parent chain's delayed inbox contract instead of sending them directly to the Sequencer. This approach is one example of a parent-to-child messaging path. More broadly, parent-to-child chain messaging covers all ways to:
```